pkg/rancher-components/src/components/Form/Checkbox/Checkbox.vue

```diff
@@ -45,7 +46,7 @@ export default {
     },
 
     valueWhenTrue: {
-      type:    null,
+      type:    [Boolean, String, Number],
```

Comment:
@richard-cox I think this covers our use case, but we might want to validate that `[Boolean, String, Number]` satisfies the type for this prop..

Reply:
Could we please write a ticket to remove this use of the component, together with `clicked` logic?
The component should not be involved in logic from the view.

Reply:
Opened #6086 so we can start collecting info for what we need to change